docs/specs/code/SERVER_STATUS_ENDPOINT.md

```diff
@@ -0,0 +1,76 @@
+# Abstract
+
+Describe a new endpoint to notify server issues to Yoroi-frontend, describe the front-end changes to handle and display this information.
+
+# Motivation
+
+Currently, Yoroi-frontend has no way to notify neither server issues nor connection issues. A new endpoint notifying the server status can be implemented in the back-end and the front-end can get this information periodically to display information if a problem is reported. Also, if the endpoint doesn't respond, the front-end can assume there is a connection issue.
+
+Also, once Yoroi has the ability to notify server issues, the endpoint used by Yoroi to fetch the information can be used to create a public URL that reports the mentioned status, such as [AWS](https://status.aws.amazon.com/) and [Twitter](https://status.twitterstat.us).
+
+# Proposal
+
+## Backend
+
+Adding a new endpoint in the backend named `api/status` is proposed. This endpoint has the method `GET` that returns an object with the form
+
+```
+{ apiStatus: true | false }
+```
+
+Where `apiStatus` is the value of the last `status` inserted in the following table:
+
+```
+CREATE TABLE yoroi_metadata.api_statuses ( id      serial    NOT NULL PRIMARY KEY
+                         , status  boolean      NOT NULL
+                         , time    timestamp with time zone NULL
+                         );
+```
+
+In this table, `id` is the index of the inserted status, `status` is a boolean that is `true` when the server reports no issue and `false` otherwise; and `time` is the time of the insertion of the mentioned status.
+
+The element insertion will be performed manually into the database. This is to simplify the implementation due to time constraints and avoid the use of a protected `POST` method (adding this is out the scope of the present spec and could be added in a future enhancement).
+The schema creation and table creation will be performed manually too, and it has to be performed before merging the front-end code and the back-end code. Once created te table must be initializated with the value
```

Comment:
this is missing the comment that I made about updating the table creation script for new deployments